packages/obojobo-sections-assessment/viewer-component.js

```diff
@@ -56,7 +56,7 @@ class Assessment extends React.Component {
 		return 'pre-test'
 	}
 
-	componentWillReceiveProps() {
+	UNSAFE_componentWillReceiveProps() {
```

Comment:
Could this be replaced with componentDidUpdate? That's the recommended React Method